### PR TITLE
Fix codestart

### PR DESCRIPTION
**Pull request by ia3andy**

### CODE REVIEW

File: roq/runtime/src/main/codestarts/quarkus/roq-codestart/base/content/posts/2024-10-20-my-first-post.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+layout: :theme/post
+title: "My First Roq Post!"
+description: This is my first article ever made with Quarkus Roq
+img: blog.avif
+tags: blogging
+author: qlover
```
Comment (mathecruz) [inline roq/runtime/src/main/codestarts/quarkus/roq-codestart/base/content/posts/2024-10-20-my-first-post.md]:
```suggestion
author: quarkusio
```

Comment (mathecruz) [reply roq/runtime/src/main/codestarts/quarkus/roq-codestart/base/content/posts/2024-10-20-my-first-post.md]: qlover is a github user haha

Comment (mathecruz) [reply roq/runtime/src/main/codestarts/quarkus/roq-codestart/base/content/posts/2024-10-20-my-first-post.md]: We need to change in `data.yml` too.

Comment (ia3andy) [reply roq/runtime/src/main/codestarts/quarkus/roq-codestart/base/content/posts/2024-10-20-my-first-post.md]: ok

File: theme/default/src/main/resources/templates/layouts/roq-default/main.html [hunk 2]
```diff
@@ -13,9 +13,7 @@
       </div>
       {/if}
       <div class="author-name">{site.data.name}</div>
-      <p>{site.description}</p>
-      <p>I crafted this blog myself and <a href="https://github.com/quarkiverse/quarkus-roq/actions/workflows/deploy-blog.yml" target="_blank">published it</a> on GitHub Pages. The sources of this example website are located <a href="https://github.com/quarkiverse/quarkus-roq/tree/main/blog" target="_blank">there</a>.</p>
-
+      <p>{site.description.markdownify.safe}</p>
```
Comment (ia3andy) [inline theme/default/src/main/resources/templates/layouts/roq-default/main.html:16]: @mcruzdev markdownify :)

File: roq/runtime/src/main/codestarts/quarkus/roq-codestart/base/content/posts/2024-10-20-my-first-post.md (hunk 1 above)
Comment (ia3andy) [reply roq/runtime/src/main/codestarts/quarkus/roq-codestart/base/content/posts/2024-10-20-my-first-post.md]: Fixed

File: roq/runtime/src/main/codestarts/quarkus/roq-codestart/base/data/authors.yml [hunk 3]
```diff
@@ -1,9 +1,8 @@
-qlover:
-  name: "Quarkus Lover"
-  email: "qlover@quarkuslover.io"
-  emailhash: "6e9f555c0f97757a625cd2e9e79a8255"
-  job_title: Senior Software Engineer
-  twitter: "quarkusio"
+roqqy:
+  name: "Roqqy Balboa"
```
Comment (mathecruz) [inline roq/runtime/src/main/codestarts/quarkus/roq-codestart/base/data/authors.yml:2]: 🤣

**Pull request merged**